### DISCUSSION TITLE
CLI to change the port visibility doesn't work

### DISCUSSION THREAD

**Discussion by dooriya** [community/community#64013] [Codespaces]
### Select Topic Area

Bug

### Body

Hi,

We use Codespaces for our getting-started samples (e.g. https://github.com/OfficeDev/TeamsFx-Samples/tree/v3/hello-world-tab-codespaces).  And we use the following GitHub CLI to automatically set the port visibility to public in our [VSCode task.json](https://github.com/OfficeDev/TeamsFx-Samples/blob/v3/hello-world-tab-codespaces/.vscode/tasks.json#L87):

 ```bash
 gh codespace ports visibility 53000:public -c $CODESPACE_NAME
``` 

It worked well in the past few months, but recently we found the command doesn't take effect :(.
The port visibility is still **private** after executing this command, and developers will fail to access the endpoint in as an iframe in Teams.

![image](https://github.com/community/community/assets/10163840/4973b736-1cdc-4bf2-9c57-280901196f55)
 

Currently the workaround is to manually configure the port visibility through VS Code UI. However, this approach is not ideal, as it introduces additional manual steps that we aim to avoid while maintaining a seamless F5 experience.

![image](https://github.com/community/community/assets/10163840/c815c097-fdc2-4f93-b35c-c06190d67607)

**Comment by dooriya** [comment]
Any update on this issue? This is really block our getting-started experience using codespaces.
The port visibility is till **private** after running command `gh codespace ports visibility 53000:public -c $CODESPACE_NAME`:
![image](https://github.com/community/community/assets/10163840/5e8e544d-a08d-4742-b962-67a5daa01964)

**Reply by codise** [reply]
I have 100% similar issue. Seems like VS Code does something different than Github CLI or there's some undocumented feature of gh cli.
